pydatalab/src/pydatalab/models/starting_materials.py

```diff
@@ -66,6 +66,12 @@ class StartingMaterial(Item):
 
     comment: Optional[str] = Field(alias="Comments")
 
+    synthesis_constituents: List[Constituent] = Field([])
```

Comment:
Minor, but as discussed I would refactor this with a `HasSynthesisInfo` trait that can then be used by both samples and starting materials, though we can discuss